rotkehlchen/externalapis/defillama.py

```diff
@@ -145,7 +145,8 @@ def query_current_price(
             self,
             from_asset: AssetWithOracles,
             to_asset: AssetWithOracles,
-    ) -> Price:
+            match_main_currency: bool,
+    ) -> Tuple[Price, bool]:
```

Comment:
you are changing the return type here and in many other function without adjusting the docstring. Adjust the docstring.